src/Salt/SiteBundle/Controller/CommentsController.php

```diff
@@ -161,20 +161,23 @@ public function downvoteAction(Comment $comment, UserInterface $user)
      */
     public function exportCommentAction(string $itemType, int $id, Request $request)
     {
+        $base_url = $request->server->get('HTTP_HOST');
```

Comment:
Should use the 3rd argument of `generateUrl()` (`UrlGeneratorInterface::ABSOLUTE_URL`) to create an aboslute url instead of getting the hostname from the server.